### ISSUE TITLE
[A11y]: Not visible and broken link element is available

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/whiteboard#895]
Link "Reload current directory" is available in html tree, but not visible and provide no function for user.

<img width="489" height="178" alt="Image" src="https://github.com/user-attachments/assets/d9c1d9d6-0843-4f1c-a8ee-308b5e6f430e" />

**Comment by blizzz** [comment]
Possibly also files related, like https://github.com/nextcloud/whiteboard/issues/898#issuecomment-3669201682
